src/pages/index.js

```diff
@@ -111,3 +218,31 @@ const enableValidation = (config) => {
 };
 
 enableValidation(config);
+
+/*      ---FETCH API DATA---      */
+/*  -Fetch initial data and render cards- */
+api
+  .getInitialCards()
+  .then((cards) => {
+    console.log(cards);
+    cards.forEach((card) => {
+      cardSection.addItem(createCard(card));
+    });
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+/*  -Fetch user info and set it in UserInfo-  */
+api
+  .getUserInfo()
+  .then((data) => {
+    userInfo.setUserInfo({
+      name: data.name,
+      description: data.about,
+    });
+    userInfo.setAvatarUrl(data.avatar);
+  })
+  .catch((err) => {
+    console.error(err);
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors